roles/default/kiali-deploy/tasks/main.yml

```diff
@@ -470,12 +476,22 @@
 # NOTE: there is a special value of accessible_namespaces - two asterisks ("**") means Kiali is to be given access to all
 # namespaces via a single cluster role (as opposed to individual roles in each accessible namespace).
 
+# Note that we look at deployment.cluster_wide_access to see if ClusterRoles are to be created or not.
+
+# if accessible_namespaces is "**", we must have cluster-wide-access, it is an error otherwise.
```

Comment:
Same deal, we need '**' for back compat (we should officially deprecate it) but unset is the new '**'.   I'll stop these comments now, even though there are more places...

Reply:
this is how it works. The ** is an internal implementation detail here. It would be very difficult to rip all the internal usage of ** out and get it right. Too much churn. We can make it where if the user doesn't set it, it becomes (internally) ** - that's how this is working right now. I can change the molecule tests to rip out the tests expicitly setting it to ** to make sure we test that ** is the default. But ripping out ** for the internal code is "unwise".

Reply:
OK